File: Python/Pattern_d7.py
"""
Patetrn

4 3 2 1
3 2 1
2 1
1

n = 4

"""
print("Enter the number of rows")
n=int(input())

print("Here is the pattern")


i=n
while i>=1:
    j=i
    while j>=1:
        print(j,end=" ")
        j-=1
    print("\n")
    i-=1

File: patternsJAVA/patternD7.java
/*
Pattern is :
4 3 2 1
3 2 1
2 1
1
*/
public class patternD7 {
    public static void main(String[] args) {
        int n=4;
        for (int i = n; i >=1; i--) {
            for (int j = i; j >=1; j--) {
                System.out.print(j+" ");
            }
            System.out.println();
        }
    }
}
